lib/api/lambda.js

```diff
@@ -101,6 +101,16 @@ Lambda.prototype.use = function(middleware) {
   return this;
 };
 
+/**
+ * Routes SCHEDULED EVENT requests for a given rule to the handler. Rule names will be the path
```

Comment:
Can you move this up above "use" (below "s3put") so it's adjacent to the other route handlers?